index.bs

```diff
@@ -2012,6 +2033,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain pick rate for a source with attribution scope</dfn> given a positive 32-bit integer |numTriggerStates|, a positive 32-bit integer |numEventStates|, a positive 32-bit integer |numUsedScopes|, and a double |epsilon|:
```

Comment:
Are `|numTriggerStates|` and `|numEventStates|` 32-bit or 64-bit? Where is the size defined?

Reply:
good point. done.